src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -683,6 +685,23 @@ private void ProcessGoAwayFrame(FrameHeader frameHeader)
             _incomingBuffer.Discard(frameHeader.Length);
         }
 
+        internal async Task FlushAsync() // no cancellation token: we don't want individual operation tokens applying to the whole connection
+        {
+            if (NetEventSource.IsEnabled) Trace("");
```

Comment:
Should this be an empty string?

Reply:
Yes. There's no additional data to trace here.